### ISSUE TITLE
Build script throws error: code_clipboard is not defined

### ISSUE THREAD

**Issue by Abion47** [Mermade/reslate#24]
**Bug Description**
I'm using the NPM version of reslate (**3.0.0-1**) to convert some widdershins markdown of an OpenAPI 3.0.3 API definition. When I run `npx reslate build`, I get the following output:

```
Compiled ../../slate/css/print.scss to ../../slate/css/print.css.
Compiled ../../slate/css/screen.scss to ../../slate/css/screen.css.
Error writing templates: (more in DEBUG output)
> Having trouble writing template: ../../_site/index.html

`TemplateWriterWriteError` was thrown
> ../../_includes/slate.ejs:38
    36|     <script src="slate/js/app/search.js"></script>
    37|     <% } %>
 >> 38|     <% if (code_clipboard) { %>
    39|     <script src="slate/js/app/copy.js"></script>
    40|     <% } %>
    41|     <script type="text/javascript">

code_clipboard is not defined

`ReferenceError` was thrown:
    ReferenceError: ../../_includes/slate.ejs:38
        36|     <script src="slate/js/app/search.js"></script>
        37|     <% } %>
     >> 38|     <% if (code_clipboard) { %>
        39|     <script src="slate/js/app/copy.js"></script>
        40|     <% } %>
        41|     <script type="text/javascript">

    code_clipboard is not defined
        at eval (../../_includes/slate.ejs:25:8)
        at slate (/home/****/test-api/node_modules/ejs/lib/ejs.js:682:17)
        at /home/****/test-api/node_modules/@11ty/eleventy/src/Engines/Ejs.js:50:14
        at TemplateLayout.render (/home/****/test-api/node_modules/@11ty/eleventy/src/TemplateLayout.js:152:31)
Problem writing Eleventy templates: (more in DEBUG output)
> Having trouble writing template: ../../_site/index.html

`TemplateWriterWriteError` was thrown
> ../../_includes/slate.ejs:38
    36|     <script src="slate/js/app/search.js"></script>
    37|     <% } %>
 >> 38|     <% if (code_clipboard) { %>
    39|     <script src="slate/js/app/copy.js"></script>
    40|     <% } %>
    41|     <script type="text/javascript">

code_clipboard is not defined

`ReferenceError` was thrown:
    ReferenceError: ../../_includes/slate.ejs:38
        36|     <script src="slate/js/app/search.js"></script>
        37|     <% } %>
     >> 38|     <% if (code_clipboard) { %>
        39|     <script src="slate/js/app/copy.js"></script>
        40|     <% } %>
        41|     <script type="text/javascript">

    code_clipboard is not defined
        at eval (../../_includes/slate.ejs:25:8)
        at slate (/home/****/node_modules/ejs/lib/ejs.js:682:17)
        at /home/****/test-api/node_modules/@11ty/eleventy/src/Engines/Ejs.js:50:14
        at TemplateLayout.render (/home/****/test-api/node_modules/@11ty/eleventy/src/TemplateLayout.js:152:31)
Benchmark (Data): `../../index.11tydata.js` took 49ms (8.1%)
Copied 9 files / Wrote 0 files in 0.17 seconds (v0.11.1)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! reslate@3.0.0-1 build.local: `npm run build.sass.local && cross-var npx eleventy --input=$SLATEDIR/*.md --output=$SLATEDIR/_site`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the reslate@3.0.0-1 build.local script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/****/.npm/_logs/2020-12-16T06_02_05_843Z-debug.log
```

The OpenAPI YAML code is as follows:

```yaml
openapi: 3.0.3
info:
  title: "Test API"
  version: 0.0.1

servers:
  - url: https://api.example.com
    description: Staging Server

paths:
  /test:
    get:
      summary: Get credentials for API access
      description: Returns OAuth2 tokens upon successful validation of a given API key
      operationId: test
      requestBody:
        content:
          application/json:
            schema:
              type: object
              required:
                - apiKey
              properties:
                apiKey:
                  type: string
      responses:
        "200":
          description: Returns OAuth2 tokens
          content:
            application/json:
              schema:
                type: object
                required:
                  - accessToken
                  - expiresIn
                properties:
                  accessToken:
                    type: string
                  expiresIn:
                    type: integer
                example:
                  accessToken: "aaaaa.bbbbb.ccccc"
                  expiresIn: 36000

```

I am running Ubuntu 18.04.5 via WSL2.

**Comment by MikeRalphson** [comment]
Thanks for the detailed report, I'll have a look.

**Comment by MikeRalphson** [comment]
Should be fixed in `reslate@3.0.0-2` - thanks again for the very detailed bug report.
